### PR TITLE
mrc-2376 Add parameters config to metadata endpoint

### PR DESCRIPTION
**Pull request by EmmaLRussell**
- parameter config returned as jsonata, to be transformed in front end - default phases need to be calculated from today. 
- doesn't include the tricky future/Vaccination Introduction parameter
- does include `transform` value for parameters which need to be scaled from % - I'm thinking we'll do this with jsonata in the form (we'll need to transform `future` into an object - the `evaluate` method I remembered from MINT is just for numerics)

Also removes some proof of concept backend code.

Not sure what code coverage is unhappy about - my attempt to cover the partial line shown by testing cast error case doesn't seem to have helped.

### CODE REVIEW

File: src/app/src/main/resources/metadata/parameterGroups/rt.jsonata [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[
+    {"start": $now(), "value": 1.18),
+    {"start": $fromMillis($millis() + (1000 * 60 * 60 *24 * 184)), "value": 2.45},
```
Comment (EmmaLRussell) [inline src/app/src/main/resources/metadata/parameterGroups/rt.jsonata]: First default phase starts today, second starts 184 days from today

**Comment by mwoodbri** [comment]
> Not sure what code coverage is unhappy about - my attempt to cover the partial line shown by testing cast error case doesn't seem to have helped.

I don't understand this either. I didn't get very far with the alternative i.e. trying to tell Jackson to deserialise as an list of ObjectNodes so that the cast was unnecessary.

More generally, some coverage has been lost because proportion of untested code has increased due to deletion of some tested code: in you were inclined this could probably be recovered by adding some tests for [httpStatusFromCode()](https://app.codecov.io/gh/mrc-ide/comet/compare/11/tree/src/app/src/main/kotlin/org/imperial/mrc/comet/clients/APIClient.kt#D1L48).

**Review by LekanAnni — APPROVED** [review]
LGTM 

I don't really have a critical point of view in terms of the app at the moment but it does look good.

**Review by mwoodbri — APPROVED** [review]
Added a comment about coverage but shouldn't prevent merging

**Pull request merged**